wordpress/README.md

```diff
@@ -57,6 +64,7 @@ WORDPRESS_SECURE_AUTH_SALT
 WORDPRESS_LOGGED_IN_SALT
 WORDPRESS_NONCE_SALT
 
+# FIXME What is the meaning of those two vars?
```

Comment:
I think these are leftovers of the original script (that I sent Krispin). The logic checking these can probably be (mostly) removed.

Reply:
"Probably" won't help me ;)
If someone can explain me the purpose of the whole if-block I could clean it up or even remove it.

Reply:
It once looked like this: https://gist.github.com/jzaefferer/3f19f62a537fdbec486af12cce16a3f7

I think these two can be removed, and the `if` reduced to this:
```
if $(wp core is-installed); then
  /bin/bash -c "/setup.sh"
  apache2-foreground
fi
```
Or if running setup.sh works while the server is running, reducing downtime:
```
if $(wp core is-installed); then
  apache2-foreground &
  /bin/bash -c "/setup.sh"
  wait
fi
```

Reply:
Done. I think a last look and the thing can be merged.